monkey/monkey_island/cc/services/reporting/issue_processing/exploit_processing/processors/exploit.py

```diff
@@ -0,0 +1,21 @@
+from __future__ import annotations
+
+from dataclasses import dataclass
+
+from monkey_island.cc.services.node import NodeService
+
+
+@dataclass
+class ExploiterReportInfo:
+    machine: str
+    ip_address: str
+    type: str
```

Comment:
Some processors extend this class, maybe should define separate dataclasses for each extension

Reply:
Dataclasses can be inherited freely so why not :)